src/runtime/renderscript_dev.ll

```diff
@@ -0,0 +1,29 @@
+target datalayout = "e-m:e-p:32:32-i64:64-v128:64:128-a:0:32-n32-S64"
```

Comment:
Well, but Renderscript engine on the device doesn't know whether code will run on GPU or CPU. It passes bitcode stream to on-device GPU "driver", which tries parsing and processing kernel bitcode. If it is successful, then code runs on GPU, if it fails, code runs on CPU.